Classes/YTPlayerView.h

```diff
@@ -51,6 +52,17 @@ typedef NS_ENUM(NSInteger, YTPlayerError) {
     kYTPlayerErrorUnknown
 };
 
+/** Different callback handlers used in JS calls to the web view. They return different kind of objects/primitives. */
```

Comment:
I think we can simplify this comment to:
"Completion handlers for player API calls"

Reply:
done